install/Cargo.toml

```diff
@@ -31,16 +31,16 @@ serde_yaml = { workspace = true }
 serde_yaml_08 = { package = "serde_yaml", version = "0.8.26" }
 solana-clap-utils = { workspace = true }
 solana-config-program = { workspace = true }
-solana-hash = { workspace = true }
-solana-keypair = { workspace = true }
-solana-logger = { workspace = true }
-solana-message = { workspace = true }
-solana-pubkey = { workspace = true }
+solana-hash = "=2.2.1"
+solana-keypair = "=2.2.1"
+solana-logger = "=2.3.1"
+solana-message = "=2.2.1"
+solana-pubkey = { version = "=2.2.1", default-features = false }
 solana-rpc-client = { workspace = true }
 solana-sha256-hasher = { workspace = true }
```

Comment:
I missed this last time, but this crate comes from the SDK and should be pinned

Reply:
yeah, the wip robot noticed too :sweat_smile: 
```
These crates have the wrong version in '/scratch/code/solana/solana/install/Cargo.toml':
    solana-sha256-hasher: has 'workspace' but expected '=2.2.1'
```

will fix this one in master and bp separately as well